lib/phoenix_live_view.ex

```diff
@@ -217,6 +217,13 @@ defmodule Phoenix.LiveView do
       this option will override any layout previously set via
       `Phoenix.LiveView.Router.live_session/2` or on `use Phoenix.LiveView`
 
+    * `:auto_connect` - if false, instructs the LiveView JavaScript client
+      to not automatically connect to the server on dead render.
+      This is useful when you have a static page that does not require
+      any connected functionality, but should render over the existing
+      connection when navigating from an already connected LiveView.
+      Defaults to `true`.
```

Comment:
And what happens if auto_connect is false and I go to a page also with auto_connect=false, trying to use patch or navigate? I assume it does a full page navigation anyway? We should document this.

Reply:
Yes. Each link will perform a full page reload.